test/framework/cluster.go

```diff
@@ -912,6 +929,60 @@ func (e *ClusterE2ETest) InstallCuratedPackage(packageName, packagePrefix string
 	})
 }
 
+func (e *ClusterE2ETest) InstallCuratedPackageFile(packageFile, kubeconfig string, opts ...string) {
+	os.Setenv("CURATED_PACKAGES_SUPPORT", "true")
+	os.Setenv("KUBECONFIG", e.kubeconfigFilePath())
+	e.T.Log("Installing EKS-A Packages file", packageFile)
+	e.RunEKSA([]string{
+		"apply", "package", "-f", packageFile, "-v=9", strings.Join(opts, " "),
+	})
+}
+
+func (e *ClusterE2ETest) generatePackageConfig(ns, targetns, prefix, packageName string) []byte {
+	yamlB := make([][]byte, 0, 4)
+	generatedName := fmt.Sprintf("%s-%s", prefix, packageName)
+	if targetns == "" {
+		targetns = ns
+	}
+	ns = fmt.Sprintf("%s-%s", ns, e.ClusterName)
+	builtpackage := &packagesv1.Package{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       packagesv1.PackageKind,
+			APIVersion: "packages.eks.amazonaws.com/v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      generatedName,
+			Namespace: ns,
```

Comment:
This seems incorrect, following the logic so far for the hello package this would be creating the package file in eksa-packages instead of eksa-packages-$CLUSTER_NAME

Reply:
ns = fmt.Sprintf("%s-%s", ns, e.ClusterName)

So this sets to the namespace to `eksa-packages-$CLUSTER_NAME`

Reply:
Oh I ss you are putting here so you change have the target namespace for the package to still be eksa-packages